athenet/layers/conv.py

```diff
@@ -63,10 +62,8 @@ def image_shape(self, value):
         )
         self.W_shared = theano.shared(W_value, borrow=True)
 
-        b_values = np.zeros((n_filters,), dtype=theano.config.floatX)
-        self.b_shared = theano.shared(b_values, borrow=True)
-
-        self.params = [self.W_shared, self.b_shared]
```

Comment:
Network treats layer.W_shared and layer.b_shared differently now, so layer.params is not applicable anymore and would only clutter code.